server/src/mikro-orm.config.ts

```diff
@@ -32,20 +33,19 @@ class CustomNamingStrategy extends AbstractNamingStrategy implements NamingStrat
   }
 }
 
+const clientUrl = process.env.DATABASE_URL
+
 // TODO: import connection-related properties from root .env
 const mikroORMConfig: Options = {
   // debug: process.env.NODE_ENV !== 'production',
+  clientUrl,
```

Comment:
Now using `DATABASE_URL` provided by Heroku

Reply:
how will this work locally?

Reply:
Apologies for the lacklustre PR description. We all need to update our `.env` to include the field `DATABASE_URL=postgresql://loolabs:loolabs@db/clubs`